guides/v2.1/frontend-dev-guide/css-topics/css-practice.md

```diff
@@ -46,25 +46,13 @@ To do this, they added an overriding `_theme.less` file in the `app/design/front
 @button-primary__hover__border: 1px solid @color-orange-red2;
 @button-primary__hover__color: @color-white;
 
-//  Navigation
```

Comment:
Hi @abrarpathan19. I wouldn't say that the code is unnecessary. The topic describes how to implement the "grey" theme sitewide. So it's absolutely applicable for the header section as well.
The problem is the provided screenshot does not reflect these changes. If we want to introduce an improvement here, we need to provide more detailed screenshot rather than removing the code that is not reflected on the screenshot.

Reply:
Hello @rogyar,
Thank for the review, Yes, I agree we wouldn't say that the code is unnecessary here but as per provided screenshots, Navigation and Header Icons are not available. that's why I removed that code. 

It enough to explaining user regarding the changing theme color scheme "Color nesting, Buttons,  Tabs,  Forms" 

Thank you !!!